docs/RELEASE_NOTES.md

```diff
@@ -34,6 +34,12 @@
 [Javadoc](https://commercetools.github.io/commercetools-sync-java/v/1.5.0/) | 
 [Jar](https://bintray.com/commercetools/maven/commercetools-sync-java/1.5.0)
 
+- 🐞 **Bug Fixes** (1)
+    - **Commons** -  Fixed a bug in the `CustomUpdateActionUtils#buildNewOrModifiedCustomFieldsUpdateActions` which caused to generate unnecessary update actions for null custom field values. [#428](https://github.com/commercetools/commercetools-sync-java/issues/428)
+
+- 🛠️ **Enhancements** (1)
+    - **Commercetools-jvm-sdk** - Upgraded to version 1.46.0 which includes a PriceCustomField serialization bug [#430](https://github.com/commercetools/commercetools-sync-java/issues/430)
```

Comment:
```suggestion
    - **Commons** - Upgraded commercetools-jvm-sdk to version [1.46.0](http://commercetools.github.io/commercetools-jvm-sdk/apidocs/io/sphere/sdk/meta/ReleaseNotes.html#v1_46_0) which includes a PriceCustomField serialization bug. [#430](https://github.com/commercetools/commercetools-sync-java/issues/430)
```

Reply:
60bf8bbcef9744ef1cf8d94e1169d5bf64186889

Reply:
It seems you didn't address this correctly 😢

Reply:
1060c4a